api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/inference/keras/KerasConstants.kt

```diff
@@ -105,6 +105,7 @@ internal const val ACTIVATION_LOG_SOFTMAX: String = "log_softmax"
 internal const val ACTIVATION_EXP: String = "exponential"
 internal const val ACTIVATION_MISH: String = "mish"
 internal const val ACTIVATION_HARDSHRINK: String = "hardshrink"
+internal const val ACTIVATION_LISHT: String = "lisht"
```

Comment:
As I understand it should be equals to function name here: https://github.com/tensorflow/addons/blob/master/tensorflow_addons/activations/lisht.py#L22

Reply:
I suppose yes, it should work for serialization/deserialization needs